api/graphql/schema.graphql

```diff
@@ -165,10 +162,149 @@ input UpdateUserInput {
     preferredLang: Language @rename(attribute: "preferred_lang")
 }
 
+input LocalizedStringInput {
+    en: String
+    fr: String
+}
+
+input UserBelongsTo {
+    create: CreateUserInput
+    connect: ID
+}
+
+input PoolBelongsTo {
+    connect: ID!
+}
+
+input CreateClassificationInput {
+    name: LocalizedStringInput
+    group: String!
+    level: Int!
+    minSalary: Int @rename(attribute: "min_salary")
+    maxSalary: Int @rename(attribute: "max_salary")
+}
+
+input ClassificationBelongsToMany {
+    sync: [ID!]
+}
+
+input CreateCmoAssetInput {
+    key: String!
+    name: LocalizedStringInput!
+    description: LocalizedStringInput
+}
+
+input CmoAssetBelongsToMany {
+    sync: [ID!]
+}
+
+input CreateOperationalRequirementInput {
+    key: String!
+    name: LocalizedStringInput!
+    description: LocalizedStringInput
+}
+
+input OperationalRequirementBelongsToMany {
+    sync: [ID!]
+}
+
+input CreatePoolCandidateInput {
+    pool: PoolBelongsTo! @belongsTo
+    user: UserBelongsTo! @belongsTo
+    cmoIdentifier: ID @rename(attribute: "cmo_identifier")
+    expiryDate: Date
+    isWoman: Boolean @rename(attribute: "is_woman")
+    hasDisability: Boolean @rename(attribute: "has_disability")
+    isIndigenous: Boolean @rename(attribute: "is_indigenous")
+    isVisibleMinority: Boolean @rename(attribute: "is_visible_minority")
+    hasDiploma: Boolean @rename(attribute: "has_diploma")
+    languageAbility: LanguageAbility @rename(attribute: "language_ability")
+    locationPreferences: [WorkRegion] @rename(attribute: "location_preferences")
+    acceptedOperationalRequirements: OperationalRequirementBelongsToMany
+    expectedSalary: [SalaryRange] @rename(attribute: "expected_salary")
+    expectedClassifications: ClassificationBelongsToMany
+    cmoAssets: CmoAssetBelongsToMany
+    status: PoolCandidateStatus
+}
+
+input PoolCandidateHasMany {
+    create: [CreatePoolCandidateInput!]
+}
+
+input CreatePoolInput {
+    owner: UserBelongsTo! @rename(attribute: "user")
+    name: LocalizedStringInput
+    description: LocalizedStringInput
+    classifications: ClassificationBelongsToMany
+    assetCriteria: CmoAssetBelongsToMany
+    essentialCriteria: CmoAssetBelongsToMany
+    operationalRequirements: OperationalRequirementBelongsToMany
+    poolCandidates: PoolCandidateHasMany
+}
+
+input UpdateClassificationInput {
+    name: LocalizedStringInput
+    group: String!
+    minSalary: Int @rename(attribute: "min_salary")
+    maxSalary: Int @rename(attribute: "max_salary")
+}
+
+input UpdateCmoAssetInput {
+    key: String!
+    name: LocalizedStringInput!
+    description: LocalizedStringInput
+}
+
+input UpdateOperationalRequirementInput {
+    key: String
+    name: LocalizedStringInput!
+    description: LocalizedStringInput
+}
+
+input UpdatePoolCandidateInput {
```

Comment:
After talking to Jerbo, seems we do want to include nested mutation for `user` here, but it should only allow the `update` operation.